### PR TITLE
GIVpower lockup details and boost modals UI

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

**Review by MohammadPCh — COMMENTED** [review]
I created a PR to your branch, please first merge it to your branch then we can approve and merge this branch.
If you want to know about my PR, it fixes some conflicts and removes files that changes unnecessary

File: next.config.js [hunk 1]
```diff
@@ -25,9 +25,9 @@ const moduleExports = withBundleAnalyzer({
 	compiler: {
 		// ssr and displayName are configured by default
 		styledComponents: true,
-		removeConsole: {
-			exclude: ['error'],
-		},
+		// removeConsole: {
+		// 	exclude: ['error'],
+		// },
```
[the remark this answers is not in the document]
Comment (MohammadPCh) [reply next.config.js:30]: I uncomment it on my PR

File: src/components/Header/Header.sc.tsx [hunk 2]
```diff
@@ -112,6 +112,11 @@ export const HBPic = styled.img`
 	border-radius: 24px;
 `;
 
+export const HBBalanceLogo = styled(HBPic)`
+	padding: 4px;
+	background: #5326ec;
+`;
+
```
[the remark this answers is not in the document]
Comment (MohammadPCh) [reply src/components/Header/Header.sc.tsx:119]: I undo these changes in my PR

**Comment by mateodaza** [comment]
Thank you @MohammadPCh this looks fine :))

File: next.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mateodaza) [reply next.config.js:30]: I was having a build problem and updating react solved it

File: package.json [hunk 3]
```diff
@@ -10,7 +10,9 @@
 		"start": "next start"
 	},
 	"resolutions": {
-		"styled-components": "^5"
+		"styled-components": "^5",
+		"@types/react": "17.0.2",
+		"@types/react-dom": "17.0.2"
```
[the remark this answers is not in the document]
Comment (mateodaza) [reply package.json:15]: Same answer as the other. Had to update react but I think it's no problem on develop. I'll update my local and remove cache and see

**Review by MohammadPCh — APPROVED** [review]
WOW, Thank you @mateodaza

**Pull request merged**